src/auth/__tests__/index.spec.js

```diff
@@ -160,6 +160,95 @@ describe('auth', () => {
     });
   });
 
+  describe('code exchange for native social', () => {
+    it('should send correct payload', async () => {
+      fetchMock.postOnce('https://samples.auth0.com/oauth/token', tokens);
+      expect.assertions(1);
+      await auth.exchangeNativeSocial({
+        subjectToken: 'a subject token',
+        subjectTokenType: 'a subject token type',
+      });
+      expect(fetchMock.lastCall()).toMatchSnapshot();
+    });
+
+    it('should send correct payload with optional parameters', async () => {
+      fetchMock.postOnce('https://samples.auth0.com/oauth/token', tokens);
+      expect.assertions(1);
+      await auth.exchangeNativeSocial({
+        subjectToken: 'a subject token',
+        subjectTokenType: 'a subject token type',
+        userProfile: {
+          name: {
```

Comment:
Is this the correct usage for SIWA, for example? Or is it all in a plain structure like

```
userProfile : {
  firstName: 'john',
  lastName: 'smith'
}
```

Reply:
For SIWA, according to the [API docs](https://auth0.com/docs/api/authentication#token-exchange-for-native-social), is:

> Expected parameter value will be JSON in the form of: { name: { firstName: 'John', lastName: 'Smith }}